src/components/Onboarding/Screens/config.js

```diff
@@ -23,40 +25,36 @@ export const Screens = [
     Screen: GardenTypeSelector,
   },
   { key: STEPS[1], title: 'Garden metadata', Screen: GardenMetadata },
+  { key: STEPS[2], title: 'Tokens settings', Screen: TokensSettings },
```

Comment:
i think that makes more sense to rename this `key` to `parent` what do you think?